monitor/src/test/java/bisq/monitor/TorHiddenServiceStartupTimeTests.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.monitor;
+
+
+import java.io.File;
+import java.util.Map;
+import java.util.Properties;
+
+import org.berndpruenster.netlayer.tor.NativeTor;
+import org.berndpruenster.netlayer.tor.Tor;
+import org.berndpruenster.netlayer.tor.TorCtlException;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import bisq.monitor.metric.TorHiddenServiceStartupTime;
+
+public class TorHiddenServiceStartupTimeTests {
+
+    private class DummyReporter extends Reporter {
+
+        private long result;
+
+        @Override
+        public void report(long value) {
+            result = value;
+
+        }
+
+        public long results() {
+            return result;
+        }
+
+        @Override
+        public void report(Map<String, String> values) {
+            report(Long.parseLong(values.values().iterator().next()));
+        }
+
+        @Override
+        public void report(Map<String, String> values, String prefix) {
+            report(values);
+        }
+
+        @Override
+        public void report(long value, String prefix) {
+            report(value);
+        }
+
+    }
+
+    private final static File torWorkingDirectory = new File(TorHiddenServiceStartupTimeTests.class.getSimpleName());
```

Comment:
Please declare fields together at top of class, not interspersed throughout methods.

Reply:
only the private class is declared above. Old habits. Done.